packages/manager/apps/web-office/src/pages/dashboard/consumption/Consumption.tsx

```diff
@@ -0,0 +1,278 @@
+import { useState } from 'react';
+import {
+  OdsCheckbox,
+  OdsFormField,
+  OdsSelect,
+  OdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  Area,
+  AreaChart,
+  CartesianGrid,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { format, parseISO, isValid, getDate } from 'date-fns';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
+
+import { useTranslation } from 'react-i18next';
+import { ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
+import {
+  getOfficeTenantUsageStatisticsQueryKey,
+  getOfficeUsageStatistics,
+  getOfficeUsageStatisticsQueryKey,
+  UsageStatisticsData,
+} from '@/api/usageStatistics';
+import {
+  useDateFnsLocale,
+  useOfficeServiceType,
+  useServiceInfos,
+} from '@/hooks';
+import CustomTooltip from './CustomTooltip.component';
+
+enum Period {
+  CURRENT = 'current',
+  LAST = 'last',
+  LAST_3 = 'last3',
+  LAST_12 = 'last12',
+}
+
+const periodOptions = [
+  Period.CURRENT,
+  Period.LAST,
+  Period.LAST_3,
+  Period.LAST_12,
+];
+export default function Consumption() {
+  const { t } = useTranslation(['dashboard/consumption']);
+
+  const [selectedPeriod, setSelectedPeriod] = useState('current');
+  const [showBusiness, setShowBusiness] = useState(true);
+  const [showProPlus, setShowProPlus] = useState(true);
```

Comment:
```suggestion
 const [lineChartShow, setLineChartShow] = useState({
    officeBusiness: true,
    officeProPlus: true,
  });
```
i think we can use only one state for all line of charts